pkg/servicemapping/servicemappingloader/servicemappingloader.go

```diff
@@ -251,7 +251,7 @@ func fileToServiceMapping(key string) (*v1alpha1.ServiceMapping, error) {
 func parseServiceMapping(b []byte) (*v1alpha1.ServiceMapping, error) {
 	var sm v1alpha1.ServiceMapping
 	if err := yaml.UnmarshalStrict(b, &sm); err != nil {
-		return nil, fmt.Errorf("error unmarshaling byte to service mapping: %w", err)
+		return nil, fmt.Errorf("error unmarshalling byte to service mapping: %w", err)
```

Comment:
I normally don't harmonize these two spellings, but the proposed spelling is fairly dominant in this repository.